code/modules/antagonists/pirate/pirate_gangs.dm

```diff
@@ -83,6 +86,7 @@ GLOBAL_LIST_INIT(pirate_gangs, init_pirate_gangs())
 	threat_title = "Tribute request"
 	threat_content = "This is the %SHIPNAME. The Silver Scales wish for some tribute \
 		from your plebeian lizards. %PAYOFF credits should do the trick."
+	arrival_announcement = "In a matter of moments, Your vessel's to be taken by surprise. We'd suggest yielding if We were you."
```

Comment:
```suggestion
	arrival_announcement = "Certainly, you don't deserve all of that aboard your vessel. It's going to fit us so much better."
```

it's not really a surprise if they announce the fact that they are about to board.